install/gcp/installer/util/utils.py

```diff
@@ -109,20 +109,52 @@ def _print_banner(border_symbol, edge_symbol, corner_symbol,
 
 
 def get_forseti_version():
-    """Get Forseti version from version file.
+    """Get the current Forseti version.
+
+    If the current version is a branch or a tag, we will return it. Otherwise
+    we will read the __init__ file for the latest version.
 
     Returns:
         str: The version.
     """
     version = None
+
+    # Check version by tag.
+    return_code, out, err = run_command(
+        ['git', 'describe', '--tags', '--exact-match'])
+    # The git command above will return the tag name if we checked out
+    # a tag, will throw an exception otherwise.
+    if return_code:
+        print(err)
+    else:
+        return 'tags/{}'.format(out.strip())
+
+    # Check version by branch.
```

Comment:
Can you add a comment here, that this is to allow install from development branch?
```
# Check version by branch.  Allow installs from development branches.
```